tests/model_execution/test_bash.py

```diff
@@ -926,6 +943,21 @@ def setUpClass(cls):
             shutil.rmtree(cls.KPARSE_OUTPUT_FOLDER)
         os.makedirs(cls.KPARSE_OUTPUT_FOLDER)
 
+    # =============================================================================
+    # Custom GulCalc checks
+    # =============================================================================
+    @staticmethod
+    def _get_getmodel_cmd(**args):
+        return "custom_gulcalc_command"
+
+    def test_custom_gul_summarycalc_1_partition(self):
+        self.genbash("custom_gul_summarycalc_1_output", 1, _get_getmodel_cmd=self._get_getmodel_cmd)
+        self.check("custom_gul_summarycalc_1_output_1_partition")
+    
+    def test_custom_gul_summarycalc_1_partition_chunk(self):
+        self.gen_chunked_bash("custom_gul_summarycalc_1_output", 1, _get_getmodel_cmd=self._get_getmodel_cmd)
+        self.check_chunks("custom_gul_summarycalc_1_output_1_partition", 1)
```

Comment:
I like this better than what I did. Adding the custom gulcalc tests to all test classes. Even if it only changes the outputs of the "err" tests, it's good to confirm that other outputs haven't changed.

Reply:
I hadn't spotted that these tests were only added to the "error guard" test class. That's probably fine for this piece of work but we could benefit from having tests for a custom gulcalc executable even without the error guard enabled. Should we add them to the base class instead?

With the current implementation, we've got `custom_gulcalc_log_start` and `custom_gulcalc_log_finish` added to the base test class but `_get_getmodel_cmd` only added to one subclass. Apparently it works but makes it unclear what we're testing.